src/eval-frame/components/reps/data-table-rep.jsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import ReactTable from "react-table";
+import PropTypes from "prop-types";
+import { get } from "lodash";
+import styled from "react-emotion";
+
+import "./react-table-styles.css";
+
+import { tinyRepSerializer } from "./rep-utils/tiny-rep-serializer";
+
+import DefaultRenderer from "./default-handler";
+import tinyRep from "./tiny-reps";
+
+const TableDetails = styled.div`
+  border: solid #e5e5e5;
+  border-width: 0px 1px 1px 1px;
+  padding: 5px;
+`;
+const TableDetailsMessage = styled.div`
+  color: #999;
+  font-style: italic;
+  font-family: "Open Sans", sans-serif;
+  padding-bottom: ${props => (props.pad ? "3px" : "0px")};
+`;
+
+const CellDetails = props => {
+  if (props.focusedPath) {
+    return (
+      <TableDetails>
+        <TableDetailsMessage pad>
+          {`details for ${props.focusedPath}`}
+        </TableDetailsMessage>
+        <DefaultRenderer value={get(props.value, props.focusedPath)} />
+      </TableDetails>
+    );
+  }
+  return (
+    <TableDetails>
+      <TableDetailsMessage>Click a table cell for details</TableDetailsMessage>
+    </TableDetails>
+  );
+};
+
+export default class TableRenderer extends React.Component {
```

Comment:
The `TableRenderer` has some weird mobile CSS applied where the buttons actually get _taller_ when the width is limited. I'm assuming this is because the table thinks we're on a mobile device. Any chance we can override that?

![Screen Shot 2019-04-10 at 3 55 29 PM](https://user-images.githubusercontent.com/95735/55919044-4bc03880-5ba9-11e9-8327-a928e9a67eba.png)

Reply:
ya i can take a look